src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj

```diff
@@ -53,7 +53,7 @@
     <PackageReference Include="Arcus.Security.Core" Version="1.6.0" />
     <PackageReference Include="Arcus.Security.Providers.AzureKeyVault" Version="1.6.0" />
     <PackageReference Include="Guard.NET" Version="1.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
```

Comment:
Upgrading to .NET 6 packages should be done when we migrate to .NET 6, so in a separate PR imho.

Reply:
Yes, of course, this is our Renovate. We'll do this separately, for sure.